week2/미션/index.html

```diff
@@ -0,0 +1,180 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="index.css" />
+    <link rel="stylesheet" href="reset.css" />
+    <title>daangn2</title>
+</head>
+<body>
+    <section class="post-header">
+        <div class="post-header__icon">
+            <button type="button">
+                <span class="local"> 군자동 </span>
```

Comment:
```suggestion
                <span class="local">군자동</span>
```
태그와 내용 사이에 공백은 제거해 주는것이 좋습니다!